src/lib/consensus/intf.ml

```diff
@@ -6,6 +6,8 @@ open Coda_numbers
 module type Prover_state_intf = sig
   type t [@@deriving bin_io, sexp]
 
+  val dummy_handler : Snark_params.Tick.Handler.t
```

Comment:
Can you not just use unhandled in the place you need it?

Reply:
handler for the base proof